### ISSUE TITLE
Resource Redirect After Force Delete

### ISSUE THREAD

**Issue by Broutard** [laravel/nova-issues#3591]
- Laravel Version: 8.64.0
- Nova Version: 3.29.0
- PHP Version: 7.4

### Description:
When a force delete is made on a resource, the redirectAfterDelete() method is not called.

Should work if `Laravel\Nova\Http\Controllers\ResourceForceDeleteController` return the same redirect as `Laravel\Nova\Http\Controllers\ResourceDestroyController`

```
if ($request->isForSingleResource() && ! is_null($redirect = $request->resource()::redirectAfterDelete($request))) {
    return response()->json([
        'redirect' => $redirect,
    ]);
}
```

**Comment by davidhemphill** [comment]
👋 Hi there! In order to keep this repository focused on bug reports, we auto-close feature requests and requests for help. Feel free to post your feature requests so others can discuss and add reactions. We'll keep an eye on them for later planning.

**Comment by crynobone** [comment]
Released in v3.30.0 <https://nova.laravel.com/releases/3.30.0>
